### PR TITLE
chore(developer): work around experimental JSON modules in node 🗜

### LINKED ISSUE #9272: chore(developer): using json modules triggers ExperimentalWarning in kmc 🗜
Coming out of #9253. When we launch kmc now, nodejs emits the following warning to stderr:

```
(node:19668) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
```

This is unfortunate because it will be annoying cmdline noise for kmc. While we can avoid this for Keyman Developer's kmc.cmd by using `--no-warnings`, this is not ideal, and won't work for the npm packaged version -- is there a way we can do this for npm as well?

### PR DESCRIPTION
**Pull request by mcdurdin**
Fixes #9272.

Credit to @srl295.

@keymanapp-test-bot skip

### CODE REVIEW

File: common/web/types/build.sh [hunk 1]
```diff
@@ -54,6 +42,16 @@ function copy_schemas() {
   rm -rf "$THIS_SCRIPT_PATH/src/schemas"
   mkdir -p "$THIS_SCRIPT_PATH/src/schemas"
   cp "${schemas[@]}" "$THIS_SCRIPT_PATH/src/schemas/"
+
+  # TODO: use json modules instead of this once they stablise
+  for schema in "${schemas[@]}"; do
+    local fn="$THIS_SCRIPT_PATH/src/schemas/$(basename "$schema" .json)"
+    echo 'export default ' > "$fn.ts"
+    cat "$fn.json" >> "$fn.ts"
+  done
```
Comment (srl295) [inline common/web/types/build.sh:51]: might even link to https://github.com/tc39/proposal-json-modules

Comment (srl295) [reply common/web/types/build.sh:51]: but it you make it look THIS easy they won't bother :]

**Review by srl295 — APPROVED** [review]
i regretfully approve :)

File: common/web/types/build.sh [hunk 2]
```diff
@@ -54,6 +42,16 @@ function copy_schemas() {
   rm -rf "$THIS_SCRIPT_PATH/src/schemas"
   mkdir -p "$THIS_SCRIPT_PATH/src/schemas"
   cp "${schemas[@]}" "$THIS_SCRIPT_PATH/src/schemas/"
+
+  # TODO: use json modules instead of this once they stablise
```
Comment (mcdurdin) [inline common/web/types/build.sh:46]:
```suggestion
  # TODO: use https://github.com/tc39/proposal-json-modules instead of this once it stablises
```

**Pull request merged**